src/main/java/org/ngafid/FindSpinEvents.java

```diff
@@ -266,6 +272,15 @@ public static void main(String [] args) {
 
             LOG.info("Processing spin events for fleet: " + fleetId);
 
+            try {
```

Comment:
Can remove this because those flights should already have been imported and things been calculated correctly.